### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by dsmf**
## Description

- Improve documentation

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -152,17 +152,18 @@ https://github.com/google/google-java-format/blob/master/README.md#intellij-jre-
 
 ### Create a release
 
-1. Choose a release version. Use semantic versioning! Create a respective branch e.g. `chore/prepare-release-2.6.1`.
-2. Add release notes for new version in [CHANGELOG.md](CHANGELOG.md) and [charts/irs-helm/CHANGELOG.md](charts/irs-helm/CHANGELOG.md) (e.g. https://github.com/catenax-ng/tx-item-relationship-service/pull/328)
-3. Update [COMPATIBILITY_MATRIX.md](COMPATIBILITY_MATRIX.md) (see [catena-x-environments](https://github.com/catenax-ng/tx-item-relationship-service/tree/catena-x-environments/charts/irs-environments))
-4. Create pull request and merge to main
+1. Choose a release version using [semantic versioning](https://semver.org/spec/v2.0.0.html).
+   and create a corresponding branch according to the template: `chore/prepare-release-x.x.x`.
+2. Add release notes for new version in [CHANGELOG.md](CHANGELOG.md) and [charts/irs-helm/CHANGELOG.md](charts/irs-helm/CHANGELOG.md) (e.g. https://github.com/catenax-ng/tx-item-relationship-service/pull/328).
```
Comment (ds-jhartmann) [inline CONTRIBUTING.md:157]: Tractusx example PR
```suggestion
2. Add release notes for new version in [CHANGELOG.md](CHANGELOG.md) and [charts/irs-helm/CHANGELOG.md](charts/irs-helm/CHANGELOG.md) (e.g. https://github.com/eclipse-tractusx/item-relationship-service/pull/429).
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -152,17 +152,18 @@ https://github.com/google/google-java-format/blob/master/README.md#intellij-jre-
...
+3. Update [COMPATIBILITY_MATRIX.md](COMPATIBILITY_MATRIX.md) (see [catena-x-environments](https://github.com/catenax-ng/tx-item-relationship-service/tree/catena-x-environments/charts/irs-environments)).
```
Comment (ds-jhartmann) [inline CONTRIBUTING.md:158]: No references to catenax-ng.
```suggestion
3. Update [COMPATIBILITY_MATRIX.md](COMPATIBILITY_MATRIX.md)
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -152,17 +152,18 @@ https://github.com/google/google-java-format/blob/master/README.md#intellij-jre-
...
+4. Create pull request from [release branch to main](https://github.com/eclipse-tractusx/item-relationship-service/compare/main...catenax-ng:tx-item-relationship-service:chore/prepare-release-x.x.x) and merge to main.
 5. Create Git tag for the desired release version `git tag x.x.x`
-   (the irs-helm tag will be created by the github workflow based on the version in the irs-helm changelog) 
-6. Push Git tag to repository `git push origin x.x.x`
-7. Wait for release workflow to complete
-8. Merge the automatically opened PR by github-actions bot
-9. Create pull request to eclipse-tractusx
-10. Notify about the release in IRS Matrix Chat. Template: 
+   (the irs-helm tag will be created by the github workflow based on the version in the irs-helm changelog).
+6. Push Git tag to repository `git push origin x.x.x`.
```
Comment (ds-jhartmann) [inline CONTRIBUTING.md:162]:
```suggestion
6. Push Git tag to repository `git push origin x.x.x`. (Requires eclipse committer role)
```

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

**Comment by dsmf** [comment]
Hi @ds-jhartmann, I'll close this pull request and incorporate the changes including your suggestions with the https://github.com/eclipse-tractusx/item-relationship-service/issues/393 story.

**Pull request closed without merging**